components/brave_wallet_ui/page/container.tsx

```diff
@@ -297,7 +346,14 @@ export const Container = () => {
 
             {!isWalletLocked &&
               <Route path={WalletRoutes.SendPage} exact>
-                <WalletPageWrapper hideBackground={true}>
+                <WalletPageWrapper
+                  wrapContentInBox={true}
+                  hideBackground={true}
+                  cardWidth={512}
+                  cardHeader={
+                    <PageTitleHeader title={getLocale('braveWalletSend')}/>
+                  }
+                >
```

Comment:
Lets remove the `header` from the `Send` view for now, so we don't re-introduce these bugs. We can follow up with a PR to add the `header` without any bugs. 

![image](https://github.com/brave/brave-core/assets/40611140/8317adf2-aec5-4b95-a2d0-d3675fac003f)

![image](https://github.com/brave/brave-core/assets/40611140/f0225089-3155-4aef-86ba-61e97c229f10)

Reply:
@Douglashdaniel Done here 6688534bf849b23cf23bbaf1c1ab4b3e69891149